packages/garbo/src/tasks/post/index.ts

```diff
@@ -339,11 +342,27 @@ function handleDrenchedInLava(): GarboPostTask {
   };
 }
 
+function acquireAbortFreeRun(): GarboPostTask {
+  const possibleFreeRuns = unlimitedFreeRunList.filter(
+    (i) => getAcquirePrice(i) < get("valueOfAdventure"),
+  );
+  const bestFreeRun =
+    possibleFreeRuns.length > 0
+      ? maxBy(possibleFreeRuns, getAcquirePrice, true)
+      : $item`none`;
```

Comment:
Imo rather than filtering and then doing maxby, do the maxby and then do the comparison, like this:
```suggestion
  const bestFreeRun = maxBy(unlimitedFreeRunList, getAcquirePrice, true);
  return (getAcquirePrice(bestFreeRun) < get("valueOfAdventure") ? bestFreeRun : $item.none;
```